mmv1/products/compute/ResizeRequest.yaml

```diff
@@ -0,0 +1,378 @@
+                      - !ruby/object:Api::Type::String
+                        name: "metricName"
+                        description: |
+                          The Compute Engine quota metric name.
+                      - !ruby/object:Api::Type::String
+                        name: "limitName"
+                        description: |
+                          The name of the quota limit.
+                      - !ruby/object:Api::Type::KeyValuePairs
+                        name: "dimensions"
+                        description: |
+                          The map holding related quota dimensions
+                      - !ruby/object:Api::Type::Integer
+                        name: "limit"
+                        description: |
+                          Current effective quota limit. The limit's unit depends on the quota type or metric.
+                      - !ruby/object:Api::Type::Integer
+                        name: "futureLimit"
+                        description: |
+                          Future quota limit being rolled out. The limit's unit depends on the quota type or metric.
+                      - !ruby/object:Api::Type::String
+                        name: "rolloutStatus"
+                        description: |
+                          Rollout status of the future quota limit.
+                  - !ruby/object:Api::Type::NestedObject
+                    name: "help"
+                    description: |
+                      [Output Only]
+                    properties:
+                      - !ruby/object:Api::Type::NestedObject
+                        name: "links"
+                        description: |
+                          [Output Only]
+                        properties:
+                          - !ruby/object:Api::Type::String
+                            name: "description"
+                            description: |
+                              Describes what the link offers.
+                          - !ruby/object:Api::Type::String
+                            name: "url"
+                            description: |
+                              The URL of the link.
+                  - !ruby/object:Api::Type::NestedObject
+                    name: "localizedMessage"
+                    description: |
+                      [Output Only]
+                    properties:
+                      - !ruby/object:Api::Type::String
+                        name: "locale"
+                        description: |
+                          The locale used following the specification defined at https://www.rfc-editor.org/rfc/bcp/bcp47.txt. Examples are: "en-US", "fr-CH", "es-MX"
+                      - !ruby/object:Api::Type::String
+                        name: "message"
+                        description: |
+                          The localized error message in the above locale.
+      # Status.lastAttempt
+      - !ruby/object:Api::Type::NestedObject
+        name: "lastAttempt"
```

Comment:
seems like their are missing values for tests but a lot of them have `output only` in the description.

Be sure to mark the attributes that are meant to be output only with `output: true` as well as include any attributes that aren't output into tests. Whether that's the test that's already been made or a new test that covers the rest of the missing attributes.

Reply:
Thanks. I was under the impression that putting _output: true_ at the status block would inherit all the way down to its childrens objects. Currently all non-output values have tests.